### PR TITLE
Make LIP 54 direct-sidechain-channel-ready

### LINKED ISSUE #420: Make Interoperability LIPs direct-channel-ready
## Motivation

The [direct sidechain channels LIP](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0072.md) is now merged, although it is not part of the coming release. 

## Current Specifications

Base Interoperability LIPs (43, 45, 49, 53, 54) defining the initial interoperability solution, contained no direct sidechain communication. Certain parts of the specs were drafted carefully to generalize automatically in case direct channels get enabled. However there are parts of the protocol that assume that no direct sidechain communication is allowed. Some examples are:

- Message Recovery defined in LIP 54. 
- TBA

## Proposed Specifications

At the time when the direct sidechain channels get live, the Base Interoperability LIPs should be modified to result to a consistent protocol supporting direct sidechain communication. 

## Affected LIPs

0053, 0054 and possibly 0045, 0049.

### PR DESCRIPTION
**Pull request by gkoumout**
Resolves #420

### CODE REVIEW

File: proposals/lip-0054.md [hunk 1]
```diff
@@ -452,13 +455,14 @@ def execute(trs: Transaction) -> None:
...
         # This function never raises an error.
-        if ccm.sendingChainID == getMainchainID():
+        if ccm.sendingChainID == OWN_CHAIN_ID:
             applyRecovery(trs, ccm)
-        # If the sending chain is not the mainchain, forward the CCM.
+        # Sending chain is not the own chain. 
+        # This can happen only in mainchain. We then forward the CCM.
         # This function never raises an error.
-        elif ccm.sendingChainID != getMainchainID():
+        else ccm.sendingChainID != getMainchainID():
```
Comment (ricott1) [inline proposals/lip-0054.md:465]: I think this should be elif, unless you remove the following comparison

**Review by ricott1 — CHANGES_REQUESTED** [review]
A minor comment

File: proposals/lip-0054.md (hunk 1 above)
Comment (gkoumout) [reply proposals/lip-0054.md]: Right, I think with else is better, forgot to remove the comparison. Done.

**Review by ricott1 — APPROVED** [review]

**Review by AndreasKendziorra — APPROVED** [review]

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**